File: 75-sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        a = b = 0
        for i in range(len(nums)):
            if nums[i] == 0: a+=1
            if nums[i] == 1: b+=1
        
        for i in range(len(nums)):
            if i < a: nums[i] = 0
            elif i >= a and i < a+b: nums[i] = 1
            else: nums[i] = 2
        

            

        

File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int a, b, c;
        a = b = c = 0;

             for (auto i : nums) {
            if (i == 0) {
                a++;
            } else if (i == 1) {
                b++;
            } else {
                c++;
            }
        }

        for (int i = 0; i < nums.size(); i++) {
            if (i < a) {
                nums[i] = 0;
            } else if (i < a + b) {
                nums[i] = 1;
            } else {
                nums[i] = 2;
            }
        }
    }
};